test/unit/specs/components/Identity/getIdentity/getIdentityOptionsValidator.spec.js

```diff
@@ -0,0 +1,40 @@
+import getIdentityOptionsValidator from "../../../../../../src/components/Identity/getIdentity/getIdentityOptionsValidator";
+
+describe("Identity::getIdentityOptionsValidator", () => {
+  it("Should throw an error when invalid options are passed", () => {
+    expect(() => {
+      getIdentityOptionsValidator({ key: ["item1", "item2"] });
+    }).toThrow(new Error("'key': Unknown field."));
+
+    expect(() => {
+      getIdentityOptionsValidator({
+        key1: ["item1", "item2"],
+        key2: ["item1", "item2"]
+      });
+    }).toThrow(new Error("'key1': Unknown field.\n'key2': Unknown field."));
+
+    expect(() => {
+      getIdentityOptionsValidator({ namespaces: [] });
+    }).toThrow(
+      new Error("'namespaces': Expected a non-empty array, but got ''.")
+    );
+
+    expect(() => {
+      getIdentityOptionsValidator({ namespaces: ["ACD"] });
+    }).toThrow(new Error("'namespaces[0]': Expected ECID, but got 'ACD'."));
+  });
+
+  it("should return valid options when no options are passed", () => {
+    expect(() => {
+      getIdentityOptionsValidator();
+    }).not.toThrow();
+    const validgetIdentityOptions = getIdentityOptionsValidator();
+    expect(validgetIdentityOptions).toEqual({ namespaces: ["ECID"] });
+  });
+  it("should not throw when supported namespace options are passed", () => {
+    const ECID = "ECID";
+    expect(() => {
+      getIdentityOptionsValidator({ namespaces: [ECID] });
+    }).not.toThrow();
+  });
+});
```

Comment:
Using the `describeValidation` utility should clean up some of these tests. See how it's used in https://github.com/adobe/alloy/blob/master/test/unit/specs/components/Privacy/validateSetConsentOptions.spec.js